deploy/tenant.yaml

```diff
@@ -22,7 +22,6 @@ spec:
         minCPU: 2 
         maxIops: 1024 
         minIops: 1024
-        iopsWeight: 
```

Comment:
why delete iopsWeight

Reply:
I deleted it by mistake, I'll put it back